### DISCUSSION TITLE
[Projects] What are Tracks that you can use to filter and sort projects?

### DISCUSSION THREAD

**Discussion by Jagusti** [community/community#36160] [Projects and Issues]
I noticed this recently, but can't find any info or documentation about Tracks.  

- What does `Tracks` refer to in the context of Issues and projects?
- Where do I set or derive values?
- What is the filter syntax?

![image](https://user-images.githubusercontent.com/521096/195899784-0969b0dc-8e4f-4417-b609-cdcbfd80346c.png)

**Comment by FahdSeddik** [comment]
Hey @Jagusti 
I can't seem to find the menu you have displayed here. Can you tell me where to find it?
![image](https://user-images.githubusercontent.com/62207434/195925399-3cd2d048-8e32-4cfb-a6b7-dbb238d4fb82.png)

**Reply by Jagusti** [reply]
This is a screenshot of the UI before that: 

![image](https://user-images.githubusercontent.com/521096/195980880-bf555be2-a384-4d90-b1bd-ae83a18a5f94.png)

You can **get to it from** the equivalent of https://github.com/users/USER/projects/N/views/1 where USER is your GitHub username and N is a numerical ID of a new style project you have.  

Similar is available for Table and Board layouts 

![image](https://user-images.githubusercontent.com/521096/195980946-d3ec6207-9c54-4899-aaa0-80f6689f5612.png)

**Reply by Jagusti** [reply]
(ie, this is in the Projects view, not the Issues view)

**Comment by numbsafari** [comment]
I am seeing this as well. I'm hoping they are A/B testing or pre-release of proper support for exposing task-list completion progress.
